Gen8/bdsp.py

```diff
@@ -190,15 +244,174 @@ def encounters():
             p += entry["minlv"].to_bytes(1, "little")
             p += entry["monsNo"].to_bytes(2, "little")
 
+    for pokemon in ug_encount_02:
+        ug_encount += (2).to_bytes(1, "little")
```

Comment:
General comment here: if the number you are putting is a constant just use the byte equivalent

ex: b"\x02"